### DISCUSSION TITLE
Feature request: Vim mode support

### DISCUSSION THREAD

**Discussion by octokatherine** [vslinko/obsidian-outliner#379] [Ideas]
I use Vim mode in Obsidian and it would be great if it played more nicely with Outliner. 

The most useful to me would be the following: 

**`o` (insert line below)** 
automatically add a bullet when inserting below/inside a list via a vim shortcut

**`O` (insert line above)** 
automatically add a bullet when inserting above/inside a list via a vim shortcut

**hotkeys**
the plugin hotkeys, like moving lines, don't work while in vim normal mode (they only work in edit mode)

**Comment by avishaan** [comment]
I really need to be able to move sections up and down. The insert line would be nice as well but moving sections is key and doesn't work in either insert nor normal mode

**Comment by diminutivesloop** [comment]
The plugin hotkeys work fine for me so maybe that was fixed at some point. However it would be nice if there was a more vim-native option for operating on list sections. Particularly when using the enhanced "command-a" shortcut it would be helpful if the selection was line-wise so I could move the section via the yank and delete line commands.

**Comment by greggroth** [comment]
I opened a PR to add support for `o` and `O` in vim mode: https://github.com/vslinko/obsidian-outliner/pull/524
